onnxruntime/test/fuzzing/src/test.cpp

```diff
@@ -271,7 +274,7 @@ int main(int argc, char* argv[]) {
   // Enable telemetry events
   //
   env.EnableTelemetryEvents();
-  struct RunStats run_stats = {0, 0, 0, 0, 0};
+  struct RunStats run_stats = {0, 0, 0, 0, 0, 0};
```

Comment:
consider `RunStats run_stats{};` which should initialize the members to 0
https://en.cppreference.com/w/cpp/language/value_initialization

Reply:
I don't  see creating a constructor for this struct and initializing it is through the constructor is better implementation considering that **the scope of the structure is only the file and it is only created once**.
Please explain why this change is necessary.
Is it more efficient?
What if I want to create the same structure with different initial values, say in future for a different instance?